docs/admin-guide.md

```diff
@@ -964,8 +964,8 @@ To determine the current version, follow these steps:
 ### ADF version you have deployed
 
 To check the current version of ADF that you have deployed, go to the management
-account in us-east-1. Check the CloudFormation stack output or tag of the
-`serverlessrepo-aws-deployment-framework` Stack.
+account in us-east-1 or cn-north-1. Check the CloudFormation stack
```

Comment:
Instead of adding `cn-north-1` everywhere where `us-east-1` is used, I would like to suggest we add a mapping table at the top to explain what values to use for each partition.

Otherwise we should also add the us-gov region everywhere, in case that partition is used.
For the US-Gov partition, the region they should deploy to is `us-gov-west-1`.